### ISSUE TITLE
When changing type of NR it keeps old type on other names

### ISSUE THREAD

**Issue by Kaineatthelab** [bcgov/name-examination#1089]
![Issue.png](https://images.zenhubusercontent.com/5a7899714b5806bc2bc5a0ce/207caed9-ea92-4f8d-af16-376eb1d771d8)
##### _**Detailed Description**_
When the request type of NR2796819 was changed from society to incorporation the system allowed the first choice to change from society to Inc. however the 2nd and 3rd choices stayed at society.  In NRO we would have been required to change the designation on all 3 choices.

**Comment by ericminio** [comment]
on it...

**Comment by ericminio** [comment]
<img width="473" alt="screen shot 2018-10-29 at 1 23 57 pm" src="https://user-images.githubusercontent.com/130439/47677993-e984f580-db7d-11e8-9684-f579543d65c3.png">

@Kaineatthelab something like that?

**Comment by Kaineatthelab** [comment]
It looks like it won’t work as they can choose different designations in an entity type. Ie: good company ltd. good company inc.


Kaine Sparks
Deputy Registrar |  BC Registries and Online Services<http://www.bcregistryservices.gov.bc.ca/>
Service BC
Ministry of Citizens’ Services
T: 778-698-1404 | Web: http://www.servicebc.gov.bc.ca<http://www.servicebc.gov.bc.ca/>
“Access to government services made easy”
[Innovation]

**Comment by ericminio** [comment]
@Kaineatthelab If we are to run a set of rules to validate the names against the entity type, will you please share whether it would be more consistent for you to have this in a separate tab along with conditions, conflicts, ... ?

**Comment by Kaineatthelab** [comment]
Hi Eric, 

Thank you for this, I like this solution while we look at the rules and do some more research. What is your estimate for dev in time for a change like this?

**Comment by ericminio** [comment]
I have it done on a separate branch and ready to send a pull request if you are happy with it

**Comment by ericminio** [comment]
First PR to catch on actual feature: PR https://github.com/bcgov/name-examination/pull/1097

**Comment by LJTrent** [comment]
@Kaineatthelab  Not sure why this is inprogress with Julia assigned to it.  Also, this is part of a bigger issue around edit .  Detaching edit form examination, and determining what can be edited based on the request state, allowing an edited NR dependent on state top go back into the queue to correct the jump queue issue we have created by putting edits into the hole queue.  This is done because we set an NR that is being edited to IN PROGRESS so that no other examiner can grab it.
